internal/nix/input.go

```diff
@@ -57,19 +71,28 @@ func (i *Input) Name() string {
 	return inputNameRegex.ReplaceAllString(result, "-")
 }
 
-func (i *Input) URLWithoutFragment() string {
-	u := i.URL // get copy
-	u.Fragment = ""
-	// This will produce urls with extra slashes after the scheme, but that's ok
-	return u.String()
+func (i *Input) URLForInput() string {
+	if i.IsDevboxPackage() {
+		resolved, err := i.lockfile.Resolve(i.String())
+		if err != nil {
+			panic(err)
+			// TODO(landau): handle error
```

Comment:
@mikeland86 gentle reminder to handle this panic's TODO (in case it slips through the cracks)